lib/document.js

```diff
@@ -2250,6 +2251,9 @@ Document.prototype.validate = function(pathsToValidate, options, callback) {
     options = pathsToValidate;
     pathsToValidate = null;
   }
+  if (options && typeof options.pathsToSkip === 'string') {
+    options.pathsToSkip = options.pathsToSkip.split(' ');
```

Comment:
For perf reasons, let's make this `options.pathsToSkip = options.pathsToSkip.indexOf(' ') === -1 ? [options.pathsToSkip] : options.pathsToSkip.split(' ');` . `split()` is very slow unfortunately, even if there's nothing to split.